components/openapi/src/test/scala/pl/touk/nussknacker/openapi/BaseOpenAPITest.scala

```diff
@@ -35,7 +36,7 @@ trait BaseOpenAPITest {
   }
 
   protected def parseResource(name: String): String =
-    IOUtils.toString(getClass.getResourceAsStream(s"/swagger/$name"), "UTF-8")
+    IOUtils.toString(getClass.getResourceAsStream(s"/swagger/$name"), StandardCharsets.UTF_8)
```

Comment:
Please use ResourceLoader instead of this.

Reply:
Out of scope of this change, that was supposed to be a quick fix, without doing changes in project dependencies (`ResourceLoader` is not available in this module)